sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpError.cs

```diff
@@ -16,21 +21,6 @@ internal static class AmqpError
         /// <summary>Indicates that a timeout occurred on the link.</summary>
         public static readonly AmqpSymbol TimeoutError = AmqpConstants.Vendor + ":timeout";
 
-        /// <summary>Indicates that a message is no longer available.</summary>
-        public static readonly AmqpSymbol MessageLockLostError = AmqpConstants.Vendor + ":message-lock-lost";
```

Comment:
These were all initially ported from Track One code; as I traced them, there were no references outside the mapping.  They appear to be Service Bus constructs that were ported over.

@ramya-rao-a , @conniey :  Any thoughts based on the error handling from your libraries would be appreciated.